test/types/atomic/doc-examples/AtomicExamples.chpl

```diff
@@ -0,0 +1,40 @@
+//
+// THIS TEST CASE IS AUTO-INCLUDED IN THE DOCUMENTATION
+//
+
+/* START_EXAMPLE_0 */
+var x: atomic int;
+/* STOP_EXAMPLE_0 */
+
+/* START_EXAMPLE_1 */
+var y: atomic int = 1;
+/* STOP_EXAMPLE_1 */
+
+/* 
+NOTE: We do not suggest this in the docs right now because the body of
+      the ': atomic int' operator returns an atomic by value, which is
+      deprecated behavior. If that somehow changes then this example
+      can go back in the docs.
```

Comment:
I really think you should just add `no copy return` to the cast and call it good for now, resolving https://github.com/chapel-lang/chapel/issues/27448. Otherwise this PR just creates more technical debt to be cleaned up when that PR is fixed